reactNativeProject/src/screens/ScreenOne/styles.js

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from "react-native";
+
+export default StyleSheet.create({
+  screen: {
```

Comment:
I wouldn´t use this name ofr the style of a `Text` component. Maybe you could rename it to something more appropiate